client/src/test/omtLinkProvider.test.ts

```diff
@@ -0,0 +1,62 @@
+import { DocumentLink, commands, workspace, Uri, Position, Range } from 'vscode';
+import { equal, deepEqual } from 'assert';
```

Comment:
In the root package.json, chai and its typings are listed as devDependencies. We should either use them in these tests, instead of assert, or remove them as devDependencies.

Reply:
redone those, now using expect().to.<>() syntax